_pages/en_US/sd-card-setup.md

```diff
@@ -120,7 +144,30 @@ If the test shows any other results, your SD card may be corrupted or damaged an
    <h2>macOS</h2>
 </noscript>
 
-## Section I - Formatting your SD card
+## Section I - Formatting your SD card with SD Formatter
+
+This section formats the SD card to the specifications by the SD Card Association. This can fix many issues that may occur with running homebrew applications.
+{: .notice--info}
+
+Any 64GB or larger SD cards will be formatted to `exFAT` in this process. You _must_ follow Section II to re-format to `FAT32`.
+{: .notice--danger}
+
+1. Download the latest version of [SD Formatter](https://www.sdcard.org/downloads/formatter/sd-memory-card-formatter-for-windows-download/)
+  - Accept the End User License Agreement to start the download
+1. Run `SD Card Formatter` (the `.exe` file) in the downloaded `.zip` file with Adminstrator privileges, then install the program
+  - The version number may be different, but it will install the same program
+1. Run `SD Card Formatter` from the Start Menu with Adminstrator privileges
```

Comment:
```suggestion
1. Run `Install SD Card Formatter` (the `.mpkg` file) in the downloaded `.zip` file
  - The version number may be different, but it will install the same program
1. Run `SD Card Formatter`
```